libs/map/src/lib/components/layer-switcher/stories/vl-map-layer-switcher.stories.ts

```diff
@@ -47,9 +55,8 @@ export const layerSwitcherDefault = () => {
             },
         ],
     };
-
     return html`
-        <vl-map>
+        <vl-map id="map">
```

Comment:
De eerste div in de VlMap zelf heeft al het id "map", kan dit voor problemen zorgen?

Reply:
voor zover ik kan zien niet, maar een `id` moet altijd uniek zijn in z'n DOM dus dat is zeker niet ideaal
aangepast